src/app/portlets/content-types/fields/content-type-fields-variables/content-type-fields-variables.component.spec.ts

```diff
@@ -80,7 +81,7 @@ describe('ContentTypeFieldsVariablesComponent', () => {
     });
 
     it('should load the component and create', () => {
-        spyOn(fieldVariableService, 'save');
+        spyOn(fieldVariableService, 'save').and.returnValue(of([]));
```

Comment:
If you spy on your service method make sure you don't get errors in the console of the browser while running those tests.